lib/cryptoexchange/exchanges/coindcx/services/market.rb

```diff
@@ -0,0 +1,53 @@
+module Cryptoexchange::Exchanges
+  module Coindcx
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super(ticker_url)
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Coindcx::Market::API_URL}/exchange/ticker"
+        end
+
+        def adapt_all(output)
+          output.map do |ticker|
+            pair = ticker['market'].insert(-4, '/')
```

Comment:
This approach is a bit dangerous, try something like https://github.com/coingecko/cryptoexchange/blob/master/lib/cryptoexchange/exchanges/abcc/services/pairs.rb